metabase/metabase
# Cloud Migration FE

Closes #41474

### Description

Implements the UI portion of allowing users to migrate their self-hosted instances to the cloud.

### How to verify

- Login as admin (you will need to do this from the brach, not a JAR for the time being)
- Go to Admin Settings, then select the Cloud item in the sidebar
- Try doing a migration (it will be sensitive to schema changes from other feature branches you've worked on, but having a fresh app db should result in a success)

### Demo

https://www.loom.com/share/f007a374634e45fcb409f8b5f2e4af4f?sid=56e52b1d-11bf-48a0-87f4-517392883641

### Checklist
